gitops/ckcp/base/kustomization.yaml

```diff
@@ -1,16 +1,46 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- deployment.yaml
 - namespace.yaml
-- rolebinding.yaml
-- route.yaml
+- pvc.yaml
+- certs.yaml
 - service.yaml
+- route.yaml
+- rolebinding.yaml
 - serviceaccount.yaml
+- deployment.yaml
 
-images:
```

Comment:
If you removed using custom image, then I think it make sence to remove https://github.com/openshift-pipelines/pipelines-service/tree/main/ckcp/docker/ckcp and script to build this custom image https://github.com/openshift-pipelines/pipelines-service/blob/main/ckcp/build.sh . Or we have to use another kustomize base to save this part.

Reply:
@AndrienkoAleksandr You're right. We won't be needing docker/ckcp anymore as we shift to use the official kcp image. Same goes for build.sh.

I discussed with @RAMESSESII2 to have a separate PR to clean up entire ckcp folder once this activity is completed.